src/test/resources/application-test.properties

```diff
@@ -8,4 +8,8 @@ spring.datasource.password=
 spring.jpa.hibernate.ddl-auto=create
 spring.jpa.show-sql=true
 spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
-SECRET_KEY=P0YfGdONKAXq8bHFO6IoIHZOhkPiNEeOi1dvnf+Ot9M=
\ No newline at end of file
+SECRET_KEY=P0YfGdONKAXq8bHFO6IoIHZOhkPiNEeOi1dvnf+Ot9M=
+
+# Daniel's API Key for Stripe (For Testing Purposes Only)
+STRIPE_API_KEY=sk_test_51PtEAN2L6Wj1zoxbpFcIYnB7YikSPoftqtdCLe73TuEp6rQAkROXjHVmK9KkKsruZpIt82AoyfgJiPsgTtIUfyji00g5uPSv0Z
```

Comment:
Is this your actual API key? probably just put a random assortment of characters for now? i don't think any stripe tests are implemented for now so it just needs dummy data